test/tests/security/test_api_security_passwd_resets.py

```diff
@@ -0,0 +1,133 @@
+'''
+Copyright 2015, DellEMC LLC
+
+   ScriptName: test_api_security_passwd_resets.py
+       Author: Torrey Cuthbert
+        Email: torrey.cuthbert@dell.com
+  Create Date: 07/28/2017
+
+      Purpose: This script consists of cases intended to test RackHD login functions.
+               As of date, there are three user account roles (Administrator, Operator, and ReadOnly).
+               We will first test the RackHD endpoint's ability to create an account for each of these
+               roles for both RackHD and Redfish interfaces (redfish=False).  We will then test the
+               Redfish endpoint's ability to also create Redfish and RackHD accounts (Redfish=True).
+               In both cases, all user roles will have established accounts in both endpoints and
+               will possess tokens for role base access and permission.
+
+           EX: python run_tests.py -stack 2
+                                   -config dellemc-test/config-mn/
+                                   -test tests/security/test_api_security_passwd_resets.py
+               python run_tests.py -stack vagrant -test tests/security/test_api_security_passwd_resets.py
+'''
+
+import sys
+import flogging
+import fit_common
+import exrex
+from nose.plugins.attrib import attr
+sys.path.append(fit_common.TEST_PATH + "/classes")
+from classes.administrator import Administrator
+from classes.readonly import ReadOnly
+from classes.operator import Operator
+
+# Globals
+logs = flogging.get_loggers()
+
+
+# Helper functions
+def createUsername():
+    return exrex.getone('[a-zA-Z]{1}[a-zA-Z0-9._\-]{1,}')
+
+
+@attr(regression=False, smoke=True, security=True)
+class TestCase01(fit_common.unittest.TestCase):
```

Comment:
Change **regression=false** to **regression=True**

This is a good regression test.